udata/frontend/markdown.py

```diff
@@ -100,6 +107,7 @@ def mdstrip(value, length=None, end='…'):
         value = value.split(EXCERPT_TOKEN, 1)[0]
     rendered = md(value)
     text = do_striptags(rendered)
+    text = bleach_clean(text)
```

Comment:
Seems to be working. I'm just wondering whether this should after or before the `do_striptags`

Reply:
Someone `do_striptags` would reintroduce the injection. I tested because it seemed more logical but...

Reply:
Fine to me, I was just wondering the reasonning :+1: